ui/app/rif/actions.js

```diff
@@ -73,4 +85,21 @@ function checkDomainAvailable (domainName) {
   }
 }
 
+function getDomainDetails (domainName) {
+  return (dispatch) => {
+    dispatch(actions.showLoadingIndication())
+    return new Promise((resolve, reject) => {
+        background.rif.rns.resolver.getDomainDetails(domainName, "0x0000000", (error, details) => {
+          if (error) {
+            dispatch(actions.displayWarning(error));
+            return reject(error);
+          }
+          console.log("DAFAK I'm HERE", details)
+          dispatch(actions.hideLoadingIndication());
+          return resolve(domainDescription);
```

Comment:
This should resolve to `details ` not `domainDescription ` right? That variable is not defined.

Reply:
You right, as i was just testing the plugin with the background console i didn't give importance to the UI. In last commit its fixed btw